### PR TITLE
Putting each piece on the ui

### PR DESCRIPTION
**Pull request by coughenberg**
Added the images of the pieces to the ui, they appear on top of the board and are currently pretty small. Might need to later scale up the window

### CODE REVIEW

**Comment by coughenberg** [comment]
Task linked: [CU-76ebwm Put each piece on the board](https://app.clickup.com/t/76ebwm)

**Comment by coughenberg** [comment]
Made with click up, and huh pretty cool. I like how it automatically links the clickup link

File: chess-ui/main.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from window import chessWindow
+from window import ChessWindow
  
 # Take name window here and grab the values
 
 # Then thise createWindow() remains, and uses the player names from the form to place in the window
-chessWindow.createwindow()
\ No newline at end of file
+ChessWindow.createWindow()
```
Comment (coughenberg) [inline chess-ui/main.py:6]: I forgot to change the calls to the class and method when we renamed these according to PEP8 standards

**Review by mattshoobs — APPROVED** [review]
Discussed refactoring later.

**Pull request merged**